### PR TITLE
PIL-1212-Enable Touch ID Modal and  PIL-1209-Alter App Startup

### PR DESCRIPTION
**Pull request by divyaBacancy**
// new wallet creation

https://user-images.githubusercontent.com/66996635/142986841-02c0e685-75ac-452a-903c-cefae53df40c.mov

//app refresh

https://user-images.githubusercontent.com/66996635/142986781-7b3a285a-d9ac-4619-8bd4-f43f2790f042.mov

//logout scenario

https://user-images.githubusercontent.com/66996635/142986795-4a7ae554-961b-423e-a6b9-b88db64ea3dc.mov

### CODE REVIEW

**Comment by divyaBacancy** [comment]
https://user-images.githubusercontent.com/66996635/144380147-584d31e2-70bc-4c78-99af-01e5b24c21ad.mov

**Comment by divyaBacancy** [comment]
https://user-images.githubusercontent.com/66996635/147535705-9cbaae56-068b-4581-bf1e-6f8b02ff0c78.mov

**Review by Yash-Chaubal — APPROVED** [review]
LGTM!

File: src/actions/onboardingActions.js [hunk 1]
```diff
@@ -191,6 +248,66 @@ export const setupWalletAction = (enableBiometrics?: boolean) => {
...
+    await dispatch(encryptAndSaveWalletAction(pinCode, ethersWallet, backupStatus, enableBiometrics));
+
+    logBreadcrumb('onboarding', 'walletSetupAction: dispatching saveDbAction for saving app settings');
+    dispatch(saveDbAction('app_settings', { appSettings: { wallet: +new Date() } }));
+
+    logBreadcrumb('onboarding', 'walletSetupAction: dispatching setupAddressAction');
+    await dispatch(setupAddressAction());
+
+    logBreadcrumb('onboarding', 'walletSetupAction: dispatching setupAppServicesAction');
+    await dispatch(setupAppServicesAction(privateKey));
```
Comment (IAmKio) [inline src/actions/onboardingActions.js:300]: Do you know if we actually need to `await` these `dispatch`es?

File: src/components/Modal/Modal.js [hunk 2]
```diff
@@ -73,6 +73,7 @@ type Props = {|
   useNativeDriver?: boolean,
   hideModalContentWhileAnimating?: boolean,
   style?: any,
+  noCloseOnBackdropPress?: boolean,
```
Comment (IAmKio) [inline src/components/Modal/Modal.js:76]: I think something like `backdropDismissable` is more readable?

File: src/components/SwipeButton/SwipeButton.js [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+    borderRadius: 14,
+  },
+  swiperBtnthumbIcon: {
+    borderRadius: 12,
+  },
+  swiperView: {
+    justifyContent: 'center',
+    alignItems: 'center',
+  },
+};
```
Comment (IAmKio) [inline src/components/SwipeButton/SwipeButton.js:89]: Thanks for making this a component 🙏

File: src/components/SwipeButton/SwipeButton.js [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+
+import * as React from 'react';
+import { View } from 'react-native';
+import RnSwipeButton from 'rn-swipe-button';
+
+// components
+import Icon from 'components/core/Icon';
+
+// utils
+import { fontSizes, spacing, appFont } from 'utils/variables';
```
Comment (IAmKio) [inline src/components/SwipeButton/SwipeButton.js:29]:
```suggestion
// Components
import Icon from 'components/core/Icon';

// Utils
import { fontSizes, spacing, appFont } from 'utils/variables';
```

**Review by IAmKio — CHANGES_REQUESTED** [review]
Just a few minor points but it's looking great!

**Pull request closed without merging**